Client/src/Pages/PageSpeed/CreatePageSpeed/index.jsx

```
@@ -24,124 +23,89 @@ const CreatePageSpeed = () => {
+      name: monitor.name || monitor.url,
+      type: monitor.type,
+      interval: monitor.interval * MS_PER_MINUTE,
+    };
 
+    const { error } = monitorValidation.validate(form, { abortEarly: false });
     if (error) {
       const newErrors = {};
-      error.details.forEach((err) => {
-        newErrors[err.path[0]] = err.message;
-      });
+      error.details.forEach((err) => (newErrors[err.path[0]] = err.message));
```

Biome

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)